client/src/linkManager.js

```diff
@@ -306,6 +308,30 @@ export class linkManager extends LuigiClientBase {
     return this;
   }
 
+  /**
+   * Sends options to the route. The parameters are used by the `navigate` function. Use it optionally in combination with any of the navigation functions and receive it as part of the context object in Luigi Client.
+   * @memberof linkManager
+   * @param {Object} options
```

Comment:
could you add the same thing as the previous comment here. It would be pretty helpful for users when reading the docu of it:

```
@param {Object} options navigation options
@param {Object} options.keepBrowserHistory <<explanation here>>
@param {Object} options.navSync <<explanation here>>
@param {Object} options.preventContextUpdate <<explanation here>>
```